wfl-slack-app/src/wfl-api.js

```diff
@@ -0,0 +1,98 @@
+var request = require('request');
+
+function postFoodOrder(restaurantId, callback) {
+    let options = {
+        url: process.env.DB_URI + '/food-order',
+        form: {
+            RestaurantId: restaurantId
+        }
+    }
+
+    request.post(options, function(err, res, body) {
+        let json = JSON.parse(body);
+        if (json) {
+            callback(json);
+        }
+    });
+}
+
+function getAllRestaurants(callback) {
+    let options = {
+        url: process.env.DB_URI + '/restaurant'
+    }
+    request.get(options, function(err, res, body) {
+        callback(JSON.parse(body));
+    });
+}
+
+function getFoodOrderById(id, callback) {
+    let options = {
+        url: process.env.DB_URI + '/food-order/' + id
+    }
+    request.get(options, function(err, res, body){
```

Comment:
Along those lines, Did you look at all at REST API clients which support Siren natively?  I wrote on ea long time ago called super-siren (https://www.npmjs.com/package/super-siren).  I'm not 100% sure it's still compatible with node 6.2 but it may be worth taking a look at?

Reply:
It would be nice to be able to use the actions we created in the hypermedia profiles for this project. I didn't even realize there were references we could use that could do this. Considering the restricted time frame I have, I think it might be a good idea to merge this PR when any pressing requested changes that might exist are handled and open a new PR to use this library.